pkg/tree/filetree.go

```diff
@@ -18,17 +18,26 @@ type FileTree struct {
 
 // NewFileTree creates a new FileTree instance.
 func NewFileTree() *FileTree {
+	tree := newTree()
+
+	// Initialize FileTree with a root "/" node
+	root := file.Path("/")
```

Comment:
This seems harmless, but it also makes it impossible to make an empty tree without removing nodes after creating an empty tree. Is this necessary/helpful to change? 

This also seems to counter the need for this line https://github.com/anchore/stereoscope/pull/49/files#diff-f864dd66d754b62339d46b3e43085b83b1d991c4c16730c7932cb52d6a952104R135 .

Reply:
Discussed — we think it's okay to assert that FileTree always has a root path (`/`), but the `RemovePath` logic needs to be amended so as not to allow the removal of this root path.